tensorflow_addons/custom_ops/image/BUILD

```diff
@@ -35,11 +78,11 @@ cc_binary(
     copts = [
         "-pthread",
         "-std=c++11",
-        D_GLIBCXX_USE_CXX11_ABI,
-    ],
+        "-D_GLIBCXX_USE_CXX11_ABI=0",
```

Comment:
Hmmm why remove the configurable ABI flag here?

Reply:
restored